smithy-model/src/main/java/software/amazon/smithy/model/shapes/ResourceShape.java

```diff
@@ -31,6 +31,7 @@
  */
 public final class ResourceShape extends EntityShape implements ToSmithyBuilder<ResourceShape> {
```

Comment:
is "properties" part of toNode somehow?

Reply:
Is something missing or broken?

Reply:
I didn't see where you were serializing properties in toNode

Reply:
Added to ModelSerializer.java